provider/go.mod

```diff
@@ -5,16 +5,16 @@ go 1.14
 require (
 	github.com/ahmetb/go-linq v3.0.0+incompatible
 	github.com/evanphx/json-patch v4.5.0+incompatible
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.2
 	github.com/googleapis/gnostic v0.2.0
 	github.com/imdario/mergo v0.3.8
 	github.com/onsi/ginkgo v1.12.0 // indirect
 	github.com/onsi/gomega v1.9.0 // indirect
 	github.com/pkg/errors v0.9.1
-	github.com/pulumi/pulumi/pkg/v2 v2.8.2
-	github.com/pulumi/pulumi/sdk/v2 v2.8.2
+	github.com/pulumi/pulumi/pkg/v2 v2.9.1-0.20200825190708-910aa96016cd
```

Comment:
Were you intending to pin to v2.9.0 specifically, or is this tag ok?

Reply:
There were a couple codegen fixes that landed after 2.9 was tagged, so this is pinning to the last commit.